cryptomatte_masks/cryptomatte_masks.py

```diff
@@ -174,9 +174,17 @@ def extract_mask(exr_file: OpenEXR.InputFile,
 
         if extract_id_mapping_from_manifest:
             # The object type and ID is encoded in the object's name in manifest
-            obj_type, obj_id_manifest = obj_name.split('_')
-            id_mapping[obj_name] = int(obj_id_manifest)
-            id_list.append(int(obj_id_manifest))
+            try:
+                obj_name_split = obj_name.split('_')
+                obj_id_manifest = obj_name_split[-1]
+                obj_type = '_'.join(obj_name_split[:-1])
+                if not obj_id_manifest.isdigit():
+                    raise ValueError(f'Could not get an ID from this entry in manifest. '
+                                     f'Expect format "classname_ID". Got: {obj_name}')
+                id_mapping[obj_type] = int(obj_id_manifest)
+                id_list.append(int(obj_id_manifest))
+            except ValueError as e:
```

Comment:
What case would we be hitting if we object id is not a digit? If that means we've messed something up then we should just fail hard here.

Reply:
Good point, made it a hard error in latest commit. 

There is a "default" field without an ID, I've added it to the ignore list after conferring with Ben.

Reply:
Thanks. Love to see this pushed to master and a release cut once we know it works.